<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-JSFL60MM" failures="1" tests="4" name="Test" time="11.657" errors="0" timestamp="2023-04-05T08:19:33 IST">
  <testcase classname="Test_Class.put_TC1" name="orchestrator" time="4.782"/>
  <testcase classname="Test_Class.patch_TC1" name="orchestrator" time="1.929"/>
  <testcase classname="Test_Class.get_TC1" name="orchestrator" time="2.649"/>
  <testcase classname="Test_Class.post_TC1" name="orchestrator" time="2.06">
    <failure type="java.lang.AssertionError" message="2023-04-05 expected [null] but found [2023-04-05]">
      <![CDATA[java.lang.AssertionError: 2023-04-05 expected [null] but found [2023-04-05]
at Test_Class.post_TC1.responseBodyValidator(post_TC1.java:70)
at Test_Class.post_TC1.orchestrator(post_TC1.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- orchestrator -->
</testsuite> <!-- Test -->
